corehq/blobs/zipdb.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+import base64
+from hashlib import md5
+from os.path import commonprefix, join, sep
+import zipfile
+
+from corehq.blobs import BlobInfo, DEFAULT_BUCKET
+from corehq.blobs.exceptions import BadName
+from corehq.blobs.interface import AbstractBlobDB, SAFENAME
+
+
+class ZipBlobDB(AbstractBlobDB):
+    """Blobs stored in zip file. Used for exporting a domain's blobs
+    """
+
+    def __init__(self, slug, domain):
+        self.zipname = 'export-{domain}-{slug}-blobs.zip'.format(domain=domain, slug=slug)
```

Comment:
I did it this way, so that the chances of a corrupt zip file are very small in comparison to having a file open for a very long time. zipfiles don't provide any sort of `flush` command